x-pack/test/api_integration/apis/metrics_ui/log_analysis.ts

```diff
@@ -94,25 +94,6 @@ export default ({ getService }: FtrProviderContext) => {
           expect(logEntryRateBuckets.data.bucketDuration).to.be(15 * 60 * 1000);
           expect(logEntryRateBuckets.data.histogramBuckets).to.be.empty();
         });
-
-        it('should return a NotFound error when the results index does not exist', async () => {
```

Comment:
This test case doesn't apply anymore, because we're not accessing the results via the alias name anymore. The access is handled internally by the programmatic ML api so we can't determine existence of the results index.